cmd/zk_quorum_check.go

```diff
@@ -33,9 +33,9 @@ const zkResponseServing = 3
 
 // NewZkQuorumCheck returns an initialized instance of *ComponentCheck.
 func NewZkQuorumCheck(name string) DCOSChecker {
-	return &ZkQuorumCheck{
-		Name: name,
-	}
+	check := &ZkQuorumCheck{Name: name}
+	check.urlFunc = check.getURL
```

Comment:
This is what we did for being able to test the json parsing with a random url, if you look at the httptest we needed this. more discussion https://github.com/dcos/dcos-checks/pull/5